MotorControl/src/main/kotlin/org/sagebionetworks/motorcontrol/presentation/compose/MotorControlPause.kt

```diff
@@ -0,0 +1,34 @@
+package org.sagebionetworks.motorcontrol.presentation.compose
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import org.sagebionetworks.assessmentmodel.presentation.AssessmentViewModel
+import org.sagebionetworks.assessmentmodel.presentation.compose.PauseScreenDialog
+import org.sagebionetworks.assessmentmodel.presentation.compose.PauseTopBar
+
+@Composable
+fun MotorControlPauseUi(
+    assessmentViewModel: AssessmentViewModel?,
+    timer: StepTimer? = null,
```

Comment:
Instead of passing in the StepTImer here you could have:`onPause: () -> Unit = {}` to match onUnpause